<launch>

<!-- joy_node -->
<node respawn="true" pkg="joy" type="joy_node" name="joy">
  <param name="dev" type="string" value="/dev/input/js0" />
  <param name="deadzone" value="0.12" />
  <param name="autorepeat_rate" value="25" />
</node>

<!-- joy_controler -->
<node pkg="joy_controler"   type="joy_controler_node"  name="joy_controler" >
  <rosparam command="load" ns="/joy_controler" file="$(find joy_controler)/config/ps2.yaml" />
</node>

</launch>
